### PR TITLE
Refactor options

### PR DESCRIPTION
**Pull request by notdryft**

### CODE REVIEW

File: build.gradle
Comment (notdryft) [inline build.gradle:23]: Wait for release before merging

File: .github/workflows/publish.yml
Comment (slandelle) [inline .github/workflows/publish.yml:14]: Why?

**Review by slandelle — APPROVED** [review]

**Comment by guilgaly** [comment]
See gatling/gatling-enterprise-plugin-commons#149 for teh corresponding changes in the enterprise-commons library

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy
Comment (guilgaly) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:216]: Dead doc link, we can fix it too while we're at it. 😉

**Review by guilgaly — APPROVED** [review]

**Pull request merged**
